### DISCUSSION TITLE
5.5.x branch

### DISCUSSION THREAD

**Discussion by gigermocas** [spring-projects/spring-integration#8714] [General]
Hi, I haven't seen any new releases in the 5.5.x branch in a while, is there a timeline for 5.5.19? I'm currently not able to bump to 6.0.x/6.1.x. just yet, and I was hoping that I would be able to pick up the current improvements, and something like https://github.com/spring-projects/spring-integration/pull/8700 (although this one isn't merged yet?)

**Comment by artembilan** [comment]
I have scheduled that release: September 19, 2023.

So far we have no plans to back-port that issue to `5.5.x`. We do that for critical bugs.
But if you insist, we can look into that, so feel free to comment respectively on that issue, so we don't forget.
